optimize/backend/src/main/java/io/camunda/optimize/license/CamundaLicense.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.license;
+
+import org.camunda.bpm.licensecheck.InvalidLicenseException;
+import org.camunda.bpm.licensecheck.LicenseKey;
+import org.camunda.bpm.licensecheck.LicenseKeyImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CamundaLicense {
+
+  public static final String CAMUNDA_LICENSE_ENV_VAR_KEY = "CAMUNDA_LICENSE_KEY";
+  private static final Logger LOGGER = LoggerFactory.getLogger(CamundaLicense.class);
+  private static final String UNKNOWN_LICENSE_TYPE = "unknown";
+  private boolean isValid;
+  private LicenseType licenseType;
+  private boolean isInitialized;
+
+  // @VisibleForTesting
```

Comment:
👀 whats this one

Reply:
`@ VisibleForTesting` is a Zeebe annotation that's used to mark methods that have a non-`private` level of visibility because of testing reasons

Since Optimize cannot use any zeebe dependencies, I commented it out. This class is a temporary copy of the monorepo implementation of `CamundaLicense`